### DISCUSSION TITLE
Ultimate vocal remover UVR5 - CHORUS

### DISCUSSION THREAD

**Discussion by MFMcr** [Anjok07/ultimatevocalremovergui#800] [General]
Greetings, sorry for the question but I am new to using Ultimate vocal remover UVR5.

I would like to know if it is also possible to extract the chorus from a song, apart from the voice and instrumentation.

Thank you so much.

**Comment by ghost** [comment]
Update 5.6 brings support for a new model that is supposed to do exactly what you want. You enable it in advanced settings -> vocal split
It will then run automatically after you used a split model. The output of that model is not quite reliable (yet), so dont get your hopes up too soon.

![vocsplit](https://github.com/Anjok07/ultimatevocalremovergui/assets/130250788/ff8931bd-2e32-4b4a-af1f-3f05a70ad577)

**Reply by MFMcr** [reply]
Thanks for answering.

What you mention is interesting, I'm going to try to do it.

I have noticed that some modes are more aggressive than others when it comes to separating voices, choruses, etc.

Greetings!

**Comment by gillesdegottex** [comment]
Out of curiosity, should we understand that back singing (chorus) and lead voices are separated in a second step?
Like: First, split vocals and non-vocals. Then split chorus and lead voice?

**Reply by ghost** [reply]
Yes thats how its supposed to work. Usually it gives good results on a wide variety of setups. Background singers seem to be the most consistently caught by the vocal split model. It can struggle when there are two concurrent singers in duets for example. Sometimes its also thrown off by specific types of synth sounds. Note that vocal splitting introduces an additional layer that can potentially introduce more artifacts into the last processed file in the chain. If you want to get the highest quality out of a specific input file, it can help to play around with the chain of order of the applied models, by running all the models manually instead of using automatic vocal splitting and deverbing, its not really a set-and-forget process.
If you're just playing around and/or using this to create some fun covers, it doesn't matter much.
Artifacts introduced by stem and vocal splitting, deverbing and de-crowding usually are kind of masked when the vocals get re-mixed with instrumental tracks.

**Reply by gillesdegottex** [reply]
Great! Thanks a lot for your detailed reply!

**Comment by pwongtl** [comment]
Hello everyone, 

I'm new to UVR5. I would like to learn how to use it to separate voice for music files. Where should I start to looking at?   

Cheers 
Pete

**Reply by ghost** [reply]
Try this video to get started:
https://www.youtube.com/watch?v=0N33sM2KRoc

**Reply by thompsonchris2002** [reply]
how come the Video isnt available anymore?

**Comment by pwongtl** [comment]
Thanks a lot!

Peter
________________________________

**Comment by juliacamel** [comment]
If you are trying to remove vocals from songs [audiomodify.com](audiomodify.com) is ever best and easy to use app that i ever seen in my life.
